docs/relay/relay.md

```diff
@@ -3,31 +3,51 @@ id: relay
+* Perform a NAT punch: This advanced technique directly connects to the host computer, even if it's on another network.
+* Use a [Relay server](https://docs.unity.com/relay/en/manual/relay-servers): A Relay server exists on the internet with a public-facing IP that you and the host can access. After the client and the host connect to a relay server, they can send data to each other through the Relay server.
 
-* Using a [Relay server](https://docs.unity.com/relay/en/manual/relay-servers): the server is on the Internet with a public-facing IP that you and the host can reach. After each side binds with the Relay, they can establish a connection and send data to each other via the Relay server.
+Netcode doesn't offer tools to help you punch through a NAT. However, you can use the Relay service provided by Unity Services to relay all technology based on Unity Transport, like Netcode.
 
-Netcode doesn't offer tools to help you do a successful punch through a NAT. But, Unity Services provides a Relay service that can relay all Unity Transport based technology, like Netcode.
+# How to use Relay
 
-# How does Relay work?
+To access a Relay server, the game host requires the following: 
+* An allocated relay server.
+* A [join code](https://docs.unity.com/relay/en/manual/join-codes). This code allows the host and clients to communicate through the Relay server without disclosing their IP addresses and ports directly.
 
-You need to install the Unity Relay SDK if you want to interact with the service. To install it, add `com.unity.services.relay` package to your project.
+## Enable Relay in a project
 
-The host of your game needs to request a Relay allocation first. This process reserves capacity on a Relay server that allows the clients to bind to it. After you request an allocation, you can get a [join code](https://docs.unity.com/relay/en/manual/join-codes) for the allocation. The join code is a random string that your clients will offer to the SDK to join the correct server and allocation.
+To enable and set up Relay in a project, follow the steps in [Get started with Relay](https://docs.unity.com/relay/en/manual/get-started).
 
-## How do I enable Relay for my project?
+## Test the Relay service in the Unity Editor
 
-The first thing to understand is that Relay is a Unity service. You need to add it to your organization in the Unity Dashboard (under the **Multiplayer** section). After that, link your project to your organization in the Unity Editor (**Project Settings** > **Services**).
+From Unity version 2022.3, you can test the Relay service in the editor: 
+1. Open the Network Manager’s inspector window. 
+2. Navigate to the Start Connection section
+3. Check the **Try Relay in the editor** box. 
+4. Select **Start Server** or **Start Host** to start the server or host, or enter the [join code](https://docs.unity.com/relay/en/manual/join-codes).
```

Comment:
```suggestion
4. Select **Start Server** or **Start Host** to start the server or host.
5. In the inspector, copy the [join code](https://docs.unity.com/relay/en/manual/join-codes).
6. Enter the join code in a new window running the same project.
7.  Select **Start Client**.
```

Reply:
That is clearer now!
If we apply the suggestion, let's not forget to remove the current line 32 ("5. Select Start Client.") which is not in the suggested removed lines, and let's remove the double space after "7."

Reply:
Thanks for catching that, removed that line in another suggestion.